pallets/multisig/src/lib.rs

```diff
@@ -118,6 +121,10 @@ pub type CreateProposalResult = sp_std::result::Result<u64, DispatchError>;
 pub trait Trait: frame_system::Trait + IdentityTrait {
     /// The overarching event type.
     type Event: From<Event<Self>> + Into<<Self as frame_system::Trait>::Event>;
+    /// Nax number of signers allowed in a multisig.
+    type MaxSigners: Get<u32>;
```

Comment:
This does not need to be a runtime variable. We can define this in the benchmarks file and then the weights can be a function of the length of signers being input. Since there is a limit on max weight, there will be an implied limit on max signers.